library/src/main/java/com/google/android/exoplayer2/util/Util.java

```diff
@@ -94,7 +94,7 @@
   private static final String TAG = "Util";
   private static final Pattern XS_DATE_TIME_PATTERN = Pattern.compile(
       "(\\d\\d\\d\\d)\\-(\\d\\d)\\-(\\d\\d)[Tt]"
-      + "(\\d\\d):(\\d\\d):(\\d\\d)(\\.(\\d+))?"
+      + "(\\d\\d):(\\d\\d):(\\d\\d)([\\.,]?(\\d+))?"
```

Comment:
I don't understand what the additional "?" is for. I'm not sure it's correct and the updated test passes without it. Please clarify or remove.

Otherwise this change looks good. Thanks!